G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/ExcelRunVBCodeCommand.md

```diff
@@ -0,0 +1,45 @@
+# excel.runvbcode
+
+## Syntax
+
+```G1ANT
+excel.runvbcode code ⟦text⟧
+```
+
+## Description
+
+This command runs a Visual Basic macro code in the currently active Excel instance. The code can be only for Sub procedures, not functions.
```

Comment:
"The code can be only for Sub procedures" => "The code can contain only Sub procedures"

Reply:
sure, but what about people who don't really know what a VBA's Sub is?

Reply:
```suggestion
This command runs a Visual Basic macro code in the currently active Excel instance. The code can contain only procedures (Sub blocks), not functions.
```